src/i18n/ES/grid-resources.ts

```diff
@@ -0,0 +1,76 @@
+  igx_grid_filter: 'Filtro',
+  igx_grid_filter_row_close: 'Cerrar',
+  igx_grid_filter_row_reset: 'Reiniciar',
+  igx_grid_filter_row_placeholder: 'Añadir un Valor para el Filtro',
+  igx_grid_filter_row_boolean_placeholder: 'Todos',
+  igx_grid_filter_row_date_placeholder: 'Fecha de Recogida',
+  igx_grid_filter_operator_and: 'Y',
+  igx_grid_filter_operator_or: 'O',
+  igx_grid_complex_filter: 'Filtro Complejo',
+  igx_grid_filter_contains: 'Contiene',
+  igx_grid_filter_doesNotContain: 'No Contiene',
+  igx_grid_filter_startsWith: 'Empieza Con',
+  igx_grid_filter_endsWith: 'Termina Con',
+  igx_grid_filter_equals: 'es Igual',
+  igx_grid_filter_doesNotEqual: 'No es Igual',
+  igx_grid_filter_empty: 'Vacío',
+  igx_grid_filter_notEmpty: 'No Vacío',
+  igx_grid_filter_null: 'Nulo',
+  igx_grid_filter_notNull: 'No Nulo',
+  igx_grid_filter_before: 'Antes',
+  igx_grid_filter_after: 'Despues',
+  igx_grid_filter_today: 'Hoy',
+  igx_grid_filter_yesterday: 'Ayer',
+  igx_grid_filter_thisMonth: 'Este Mes',
+  igx_grid_filter_lastMonth: 'El Mes Pasado',
+  igx_grid_filter_nextMonth: 'El Próximo Mes',
+  igx_grid_filter_thisYear: 'Este Año',
+  igx_grid_filter_lastYear: 'El Año Pasado',
+  igx_grid_filter_nextYear: 'El Próximo Año',
+  igx_grid_filter_greaterThan: 'Mayor que',
+  igx_grid_filter_lessThan: 'Menor que',
+  igx_grid_filter_greaterThanOrEqualTo: 'Mayor o Igual que',
+  igx_grid_filter_lessThanOrEqualTo: 'Menor o igual que',
+  igx_grid_filter_true: 'Verdadero',
+  igx_grid_filter_false: 'Falso',
+  igx_grid_filter_all: 'Todas',
+  igx_grid_filter_condition_placeholder: 'Seleccionar Filtro',
+  igx_grid_summary_count: 'Cuenta',
+  igx_grid_summary_min: 'Min',
+  igx_grid_summary_max: 'Max',
+  igx_grid_summary_sum: 'Sum',
+  igx_grid_summary_average: 'Promed',
+  igx_grid_summary_earliest: 'El mas temprano',
+  igx_grid_summary_latest: 'El último',
+  igx_grid_excel_filter_moving_left: 'izquierda',
+  igx_grid_excel_filter_moving_right: 'derecha',
+  igx_grid_excel_filter_moving_header: 'moviendo columna',
+  igx_grid_excel_filter_sorting_asc: 'ascendiendo',
+  igx_grid_excel_filter_sorting_desc: 'descendiendo',
+  igx_grid_excel_filter_sorting_header: 'clasificando',
+  igx_grid_excel_custom_dialog_add: 'añadir filtro',
+  igx_grid_excel_custom_dialog_clear: 'Limpiar Filtro',
+  igx_grid_excel_custom_dialog_header: 'Auto-filtro personalizado en una columna: ',
+  igx_grid_excel_cancel: 'cancelar',
+  igx_grid_excel_apply: 'aceptar',
+  igx_grid_excel_search_placeholder: 'Buscar',
+  igx_grid_excel_select_all: 'Seleccionar todos',
+  igx_grid_excel_blanks: '(Espacios en blanco)',
+  igx_grid_excel_hide: 'Ocultar Columna',
+  igx_grid_excel_pin: 'Anclar columna',
+  igx_grid_excel_unpin: 'Desanclar columna',
+  igx_grid_excel_text_filter: 'Filtro de Texto',
+  igx_grid_excel_number_filter: 'Filtro Numérico',
+  igx_grid_excel_date_filter: 'Filtro de Fecha',
+  igx_grid_excel_boolean_filter: 'Filtro Booleano',
```

Comment:
@bkulov  Should these be Title Case?
Text filter -> Filtro de Texto
...

Reply:
No. These are menu items. Only the first word should start with upper case letter and the others should be lower case.